app/controllers/profiles_controller.rb

```diff
@@ -68,175 +63,106 @@ def render_footer?
 
   def typeahead
     suggestions = Profile.typeahead(params[:q], region: current_region)
-    suggestions_array = suggestions.map do |suggestion|
-      {'text': suggestion.titleize}
-    end
+    suggestions_array = suggestions.map { |suggestion| { 'text': suggestion.titleize } }
     suggestions_array.push('_source': {})
     respond_with(suggestions_array)
   end
 
   private
 
-  # Use callbacks to share common setup or constraints between actions.
   def set_profile
     @profile = Profile.friendly.find(params[:id])
   end
 
-  # Never trust parameters from the scary internet, only allow the white list through.
   def profile_params
     params.require(:profile).permit(
-      :email,
```

Comment:
why did you organize the params like this? I found it more readable before.

Reply:
It is was more readable before, but also increases the page size a lot in length.